cmake/FindGSL.cmake

```diff
@@ -91,8 +91,7 @@ ELSE(WIN32)
       "${GSL_ROOT_DIR}/bin"
       CACHE STRING "preferred path to GSL (gsl-config)")
     FIND_PROGRAM(GSL_CONFIG gsl-config
-      ${GSL_CONFIG_PREFER_PATH}
-      /usr/bin/
```

Comment:
This change is unintentional. Disregard it if you feel that "/usr/bin" must be listed explicitly.